src/StatusQ/Popups/StatusSearchPopup.qml

```diff
@@ -0,0 +1,198 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.12
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+
+StatusModal {
+    id: root
+    width: 700
+    height: (!!searchResults && (searchResults.count >= 0) && (searchText !== "")) ? (((searchResults.count < 5)) ? 560 : 770) : 142 //970
+    anchors.centerIn: parent
+    showHeader: false
+    showFooter: false
+
+    property string searchText: contentComponent.searchText
+    property string selectionBadgeImage: ""
+    property string selectionBadgePrimaryText: ""
+    property string selectionBadgeSecondaryText: ""
+    property string selectionBadgeIdenticonColor: ""
+    property bool loading
+    property Component searchOptionsPopupMenu
+    property ListModel searchResults: ListModel { }
+    function resetSelectionBadge() {
+        selectionBadgeImage = "";
+        selectionBadgePrimaryText = "Anywhere";
+        selectionBadgeSecondaryText = "";
+        selectionBadgeIdenticonColor = "";
+    }
+
+    content: Item {
+        width: parent.width
+        height: root.height
+        property alias searchText: inputText.input.text
+
+        ColumnLayout {
+            id: contentItemColumn
+            anchors.fill: parent
+            spacing: 0
+            Item {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 63
+                StatusInput {
+                    id: inputText
+                    anchors.left: parent.left
+                    anchors.leftMargin: 8
+                    anchors.right: parent.right
+                    anchors.rightMargin: 8
+                    anchors.bottom: parent.bottom
+                    input.icon.name: "search"
+                    input.placeholderText: "Search"
+                    input.bgVisible: false
+                }
+            }
+            StatusMenuSeparator { Layout.fillWidth: true }
+            Item {
+                Layout.preferredWidth: parent.width
+                Layout.preferredHeight: 58
+                StatusButton {
+                    id: searchOptionsMenuButton
+                    anchors.left: parent.left
+                    anchors.leftMargin: 16
+                    anchors.verticalCenter: parent.verticalCenter
+                    implicitWidth: (contentItemRowLayout.width + 8)
+                    implicitHeight: 32
+
+                    contentItem: Item {
+                        anchors.fill: parent
+                        RowLayout {
+                            id: contentItemRowLayout
+                            anchors.centerIn: parent
+                            spacing: 2
+                            StatusBaseText {
+                                text: "In: "
+                            }
+                            StatusRoundedImage {
```

Comment:
In front of this image (or letter identicon) there's a possibility that we need to render another icon, in case we're selecting a private group chat type:

<img width="247" alt="Screenshot 2021-08-03 at 23 18 36" src="https://user-images.githubusercontent.com/445106/128087599-b0f75cd1-bd3b-4e19-a905-84599f75e8f4.png">

You can check  https://github.com/status-im/StatusQ/blob/master/src/StatusQ/Components/StatusChatListItem.qml#L126-L137 to see what you need for this.

Reply:
done, added